zerver/migrations/0379_message_edit_history_remove_field.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0378_message_edit_history_update_fields"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="archivedmessage",
+            name="edit_history",
+        ),
+        migrations.RemoveField(
+            model_name="message",
+            name="edit_history",
+        ),
+    ]
```

Comment:
This migration needs to be after the commit that switches to reading from the new JSON field -- generally when deploying a complex migration like this, we'll want to deploy individual commits one-by-one, and we can't delete these fields if they are still being accessed.  

(Usually, removing the old fields will be the last commit).